src/sentry/assistant/guides.py

```diff
@@ -114,7 +114,7 @@
                 'title': _('Users'),
                 'message': _(
                     'Sending user data to Sentry will unlock a number of features, primarily the ability to drill down into the number of users affected by an issue. '
-                    'Learn how easy it is to set this up today.'),
+                    'Learn how easy it is to <a href="https://docs.sentry.io/learn/context/#capturing-the-user" target="_blank">set this up </a>today.'),
```

Comment:
how about "... the ability to **see which users are affected** by an issue."? also lol @ second sentence, you sound like a salesperson.

Any reason we're using target blank here? It's generally considered bad practice.